crates/net/discv4/src/lib.rs

```diff
@@ -1554,6 +1559,11 @@ impl Discv4Service {
                             let _ = self.local_eip_868_enr.set_tcp6(port, &self.secret_key);
                         }
                     }
+
+                    Discv4Command::Terminated => {
+                        //self.terminate();
+                        todo!()
```

Comment:
this needs to emit the event 

and here we need to handle the terminate:

https://github.com/paradigmxyz/reth/blob/858ea41da1f8852e93e0604dbb2f7a89a5398b28/crates/net/discv4/src/lib.rs#L1618-L1618